datanode/sqlstore/deposits.go

```diff
@@ -145,8 +145,12 @@ func (d *Deposits) getByPartyCursorPagination(ctx context.Context, party string,
 func getDepositsByPartyQuery(party string, dateRange entities.DateRange) (string, []interface{}) {
 	var args []interface{}
 
-	query := fmt.Sprintf(`select id, status, party_id, asset, amount, foreign_tx_hash, credited_timestamp, created_timestamp, tx_hash, vega_time
-		from deposits_current where party_id = %s`, nextBindVar(&args, entities.PartyID(party)))
+	query := `select id, status, party_id, asset, amount, foreign_tx_hash, credited_timestamp, created_timestamp, tx_hash, vega_time
```

Comment:
I know I'm only supposed to look at the schema.gql but the SQL syntax between this and the next file is inconsistent and it bothers me 😂